libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -280,18 +278,17 @@ export class PaperHandlerDriver implements PaperHandlerDriverInterface {
     // According to the manual, "REAL-TIME USB PROTOCOL FORMAT" supports transferring out optional data.
     // The prototype doesn't support it, so leave out support from this function until needed.
   ): Promise<Result<T, CoderError>> {
-    await this.realTimeLock.acquire();
+    return await this.realTimeLock.withLock(async () => {
```

Comment:
As noted in the PR description, using this `withLock` method is nice because it uses a `try` `finally` under the hood to make sure we release the mutex even in error cases.

Previously, the mutex would not be released if there was an error before the `.release()` call